README.md

```diff
@@ -251,7 +251,7 @@ Based on [FHIR MedicationRequest](https://hl7.org/fhir/R4B/medicationrequest.htm
 |id|string|-|[Resource](https://hl7.org/fhir/R4B/resource.html): Logical id of this artifact|
 |medication|Reference(Medication) or CodeableConcept|-|CodeableConcept containing one of the codes from /medications/$medicationId$|
 |dosageInstruction|Dosage|-|-|
-|patient|string|e.g. "allergy", "intolerance"|`userId` as used in /users/$userId$ and related collections.|
+|patient|string|-|`userId` as used in /users/$userId$ and related collections.|
```

Comment:
We could actually also get rid of this property altogether, since that object exists in the user's own collections anyways - what do you think?

Reply:
I agree. I don't think we'll ever use `medicationRequests` outside of `user` itself, without querying users as well, so that makes sense

Reply:
okay, then let's do this right in this PR, would that make sense?

Reply:
[Remove patient field completely](https://github.com/StanfordBDHG/ENGAGE-HF-Firebase/pull/50/commits/57a978d592b8612c5c1f93604724893bf2f27f78)

I checked other places and I don't think it was referenced anywhere